README.md

```diff
@@ -27,14 +27,34 @@ or
 Usage
 -----
 
-`python -m proofreader [targets]`
-targets: being paths to python files or packages seperated by spaces
+`proofreader --targets dir file1.py file2.py`
+
+`--targets`: Paths to python files or packages seperated by spaces that you would like run through `pylint` and `flake8`.
+
+`--check-licenses`: Optional check for restricted licenses of installed packages.
+
+Example usage:
+`proofreader --targets dir file1.py file2.py --check-licenses true`
+
+or in isolation:
+
+`proofreader --check-licenses true`
 
 Configuration
 -------------
 
 `proofreader` contains a default configuration files for both [pylint](https://github.com/PyCQA/pylint) and [flake8](https://github.com/PyCQA/flake8) these can be found in `proofreader.config` these can be overridden by providing your own configuration files in the root directory you run the `proofreader` module from.
 
+To configure restricted license types simply provide a file named `.restricted_licenses.txt` and included one license name per line.
+
+Example `.restricted_licenses.txt` contents:
```

Comment:
opinions on the naming of this? Should we use `whitelist` somewhere in the name if it's a whitelist (doesn't appear to be a blacklist)?

Reply:
do we want a default file from `proofreader`, to be overridden?

Reply:
I'd err on the side of building up a white list. incompatible and modified licences tend to grow, compatible ones would be more static.

Reply:
Have changed to use a whitelist. A default is provided by the package (still allows local overrides if required).

I have not added any `whitelist` default values yet, open to suggestions, though, as mentioned,  these can be added periodically.